### DISCUSSION TITLE
Suggestion for handling prefixes + more pressure units (kPa, mbar, micron)

### DISCUSSION THREAD

**Discussion by flaskhals** [sadellie/unitto#17] [Ideas]
First of all: thanks for a great app! I use it pretty much daily and really like it. 

I have a few ideas about improvements though. First of all, the pressure unit kilopascal seems to be missing which is a bit funny considering kilo one of the more used prefixes for Pascal. 

Millibar (mbar) and micron are two other pressure units that would be nice to include. Micron is a dumb unit but it's used for example in the US HVAC trade. See:
https://hvacrschool.com/what-is-a-micron/

Lastly, to avoid clutter, having prefixes separate from units could maybe be a solution. The list of units could be kept as it is but all variants of each unit is removed and a prefix drop down-menu is added where you can select your preferred prefix.

Again: Thanks for a useful tool!

**Comment by sadellie** [comment]
Hello, thank you for the kind words.

# Missing units
Added units in 6eaaf68ae0f088c57737805807bad1970a708f35.

# Prefixes
Unfortunately, this one is difficult to implement.
1. This feature adds at least 2 additional clicks (open drop-down and choose prefix) + scroll.
2. It will break the search functionality and confuse users.
3. There are units that don't need prefixes.
4. A huge refactor will be required (UI and data) + data migration.

_**Thank you**_ for using Unitto!
